libxrdp/libxrdp.c

```diff
@@ -1153,15 +1153,6 @@ libxrdp_reset(struct xrdp_session *session,
             return 0;
         }
 
-        /* if same (and only one monitor on client) don't need to do anything */
```

Comment:
Why remove this? It doesn't appear to affect multi-mon?

Reply:
Good question!

The answer is that at the moment, when we go from two monitors to one, `client_info->multimon` gets reset. As a result this test succeeds and we don't send the signal back to the state machine that the resize has completed.

Reply:
Well then I utterly failed at making this call do what I thought it did, thanks for removing it then!